highfidelity/hifi
# Case 20562: Don't import/export local entities

https://highfidelity.manuscript.com/f/cases/20562/Export-Import-of-local-entities

Test plan:
- Create two cubes.  Export them to JSON.
- Open the JSON and add the line `"entityHostType": "local"` to one of them.
- Import the entities.  Only one (the one you didn't change) should be imported.
